etc/condor/scripts/worker.W2012Pipe.sh

```diff
@@ -18,7 +18,7 @@ module add gcc/4.4.5
 
 echo ${HERE}
 
-which gcc
+command -v gcc
```

Comment:
This is just telling the user what compiler is in the path??? Maybe in that case `type -a gcc` would be more informative?

Reply:
I assumed the intent was to document the actual compiler used.

Reply:
That's reasonable.